City-of-Helsinki/berth-reservations-admin
# VEN-650 | Add applicantion's boat information to OfferPage

## Description :sparkles:

Adds a boat information card to the offer page.

### Closes :no_good_woman:

[VEN-650](https://helsinkisolutionoffice.atlassian.net/browse/VEN-650)

## Testing :alembic:

### Automated tests :gear:️

`src/domain/offer/boatCard/__tests__/BoatCard.test.tsx `

### Manual testing :construction_worker_man:

1. Open applications page
2. Select an application with an attached boat
3. Select a harbor to offer
4. Card with attached boat details should be displayed
